### PR TITLE
ss-8: add encrypt service

### PR DESCRIPTION
**Pull request by Evcat6**

### CODE REVIEW

File: backend/src/migrations/20230215152530_add_users_table.ts
Comment (nikita-remeslov) [inline backend/src/migrations/20230215152530_add_users_table.ts:1]: Not sure that you need delete migration

File: backend/src/common/helpers/crypt/encrypt-sync/encrypt-sync.helper.ts
Comment (nikita-remeslov) [inline backend/src/common/helpers/crypt/encrypt-sync/encrypt-sync.helper.ts:1]: Create a service for it. And use it in the sign up route (check TODO comments in the code base)

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Looks good. Need to fix some comments

File: backend/src/migrations/20230215152530_add_users_table.ts
Comment (Evcat6) [reply backend/src/migrations/20230215152530_add_users_table.ts:1]: Resolved at [ad7bc18](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/31/commits/ad7bc180453acd0a8e9b51ad4dee1e5128c95d13)

File: backend/src/common/helpers/crypt/encrypt-sync/encrypt-sync.helper.ts
Comment (Evcat6) [reply backend/src/common/helpers/crypt/encrypt-sync/encrypt-sync.helper.ts:1]: Done in [1be6fc4](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/31/commits/1be6fc42cec9971d77c96859961e2ea97043eb52)

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**